server/core/src/main/java/org/infinispan/server/core/AbstractProtocolServer.java

```diff
@@ -123,6 +125,16 @@ public void start(C configuration, EmbeddedCacheManager cacheManager) {
       }
    }
 
+   @Override
+   public final void postStart() {
+      try {
+         internalPostStart();
+      } catch (RuntimeException e) {
```

Comment:
Should this be Throwable ?

Reply:
This was mimicking the `start` method above. I can change it, but we'll need to wrap it on a runtime exception.